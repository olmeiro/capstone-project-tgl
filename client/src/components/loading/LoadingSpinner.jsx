import React from 'react'
import './spinner.css'

import cohete from '../../../assets/cohete.jpeg'

export default function LoadingSpinner () {
  return (
    <div className="spinner-container">
      <div className="border-team-green">
        <img
          src={cohete}
          alt="logo team"
          className="md:w-96 md:h-96 sm:w-48 sm:h-48"
        />
      </div>
      <div className="flex justify-center">
        <div className="loading-spinner "></div>
      </div>
    </div>
  )
}
