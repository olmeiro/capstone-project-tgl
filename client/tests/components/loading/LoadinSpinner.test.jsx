import { render, screen } from '@testing-library/react'
import LoadingSpinner from '../../../src/components/loading/LoadingSpinner'

describe('test on <LoadingSpinner />', () => {
  test('should match to Snapshot', () => {
    const {container } = render(<LoadingSpinner />)
    screen.debug()

    expect(container).toMatchSnapshot()
  })
})
