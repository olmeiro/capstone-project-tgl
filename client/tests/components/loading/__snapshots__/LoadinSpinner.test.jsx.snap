// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test on <LoadingSpinner /> should match to Snapshot 1`] = `
<div>
  <div
    class="spinner-container"
  >
    <div
      class="border-team-green"
    >
      <img
        alt="logo team"
        class="md:w-96 md:h-96 sm:w-48 sm:h-48"
        src=""
      />
    </div>
    <div
      class="flex justify-center"
    >
      <div
        class="loading-spinner "
      />
    </div>
  </div>
</div>
`;
